KatherineK/comparelists.py

```diff
@@ -0,0 +1,15 @@
+a = [1,2,3]
+b = [1,2,3]
+same = 0
+
+if len(a) != len(b) :
```

Comment:
Seems there is some code duplication in this file. I bet you could figure out a way to optimize your logic and make it to where you don't have to write the same line of code twice.